modules/lib/src/main/resources/assets/js/app/ai/AI.ts

```diff
@@ -238,6 +241,26 @@ export class AI {
         }
     };
 
+    private handleInteractionEvent = (event: AiContentOperatorInteractionEvent) => {
+        if (event.interaction === 'click') {
```

Comment:
It's better to have switch here, because that's what you do, right? If you have switch without the `default` clause, it will show error, if you add new interaction, but forget to add the handler. So it's kinda handy, IMHO.